src/components/Home/HomeFigure.tsx

```diff
@@ -88,7 +88,12 @@ const MainImage = styled.img`
 	`}
 `;
 
-const Button = styled.div<{ pointX: number; pointY: number }>`
+interface point {
+  pointX: number;
+  pointY: number;
+}
+
+const Button = styled.div<point>`
```

Comment:
```suggestion
interface Point {
  pointX: number;
  pointY: number;
}

const Button = styled.div<Point>`
```
타입명, 인터페이스명, 클래스명 등은 Pascal Case 로 작성하여주세요.

const point: Point = { pointX: 10, pointY: 20 };

과 같이 변수명 과 구별되기 위해서 그렇게 합니다.

Reply:
넵! 리뷰해주신대로 수정해서 머지 하겠습니다 감사합니다~!